.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -0,0 +1,147 @@
+name: Publish CDK Manually
+on:
+  push:
+    paths:
+      - '**.yml'
+    branches:
+      - issue-19981_avoid-race-condition-on-airbyte-cdk-release
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: "Repo to check out code from. Defaults to the main airbyte repo. Set this when building connectors from forked repos."
+        required: false
+        default: "airbytehq/airbyte"
+      gitref:
+        description: "The git ref to check out from the specified repository."
+        required: false
+        default: master
+      part-to-bump:
+        type: choice
+        description: "Choose the type of version upgrade : major|minor|patch"
+        options:
+          - major
+          - minor
+          - patch
+        required: true
+      dry-run:
+        description: 'By default dry-run publishes to Test PyPi. Use "false" to publish to actual PyPi servers.'
+        required: false
+      changelog:
+        description: "Changelog message to be added to CHANGELOG.md"
+        required: true
+
+concurrency:
+  group: publish-airbyte-cdk
+  cancel-in-progress: false
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.event.inputs.repo }}
+          ref: ${{ github.event.inputs.gitref }}
+      - name: "Publish Airbyte CDK: bump version"
+        run: |
+          pip install bumpversion
+          cd airbyte-cdk/python
+          bumpversion patch
+          new_version="$(grep -i 'current_version = ' .bumpversion.cfg | sed -e 's/.* = //')"
+          awk -v NEW_VERSION="$new_version" -v CHANGELOG_MESSAGE="This is a changelog message for testing purposes" 'NR==3{print "## " NEW_VERSION "\n" CHANGELOG_MESSAGE "\n"}1' CHANGELOG.md > tmp && mv tmp CHANGELOG.md
+      - name: Commit and Push Changes
+        if: success()
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          file_pattern: airbyte-cdk/python/setup.py airbyte-cdk/python/.bumpversion.cfg airbyte-cdk/python/CHANGELOG.md
+          commit_message: 🤖 Bump patch version of Airbyte CDK
+          commit_user_name: Octavia Squidington III
+          commit_user_email: octavia-squidington-iii@users.noreply.github.com
+      - name: Publish Failures to issue-19981_test-notification
```

Comment:
Note to myself: to be confirmed with the team. If we decide this is what we want, we need to:
* choose the channel (right now, this channel id is the one from #issue-19981_test-release-notification)
* add Airbyte Tooling Bot to the channel

Reply:
I'm good with a designated slack channel for cdk releases. Altertnatively we could try to retain the same behavior and just post a comment at the end of the GH issue. The only difference being that now it would be after the merge

Reply:
I think we get a lot of value without notifications at first. Then we can add notifications as a follow up to this PR.

Reply:
Posting a comment at the end of a GH issue will be challenging since this new way of releasing makes it that there is no GH issue associated with the release (at least, not that the GitHub workflow knows about)